crates/ruff_cli/tests/integration_test.rs

```diff
@@ -1100,6 +1105,7 @@ fn unreadable_dir() -> Result<()> {
     success: true
     exit_code: 0
     ----- stdout -----
+    Found 0 errors in 1 file
 
     ----- stderr -----
     warning: Encountered error: Permission denied (os error 13)
```

Comment:
Flagging here that we're displaying the `Found 0 errors in 1 file`, even when we run into a permission-denied error. Perhaps the output here is undesirable?

Open to feedback and suggestions on what we think the ideal product behavior should look like!